src/state/profile/hooks.ts

```diff
@@ -55,10 +34,10 @@ export const useGetProfileAvatar = (account: string) => {
       dispatch(fetchProfileAvatar(account))
     }
 
-    if (!username && nft && address) {
+    if (!username && address) {
```

Comment:
Users might remove their profile picture but still have username, I think this should not check for `nft`.
There is a drawback that for unregistered users it will spam console with 404 errors, but I think showing as many usernames as possible is more important 🤔

Reply:
Is there another datapoint that could be added to the `profileAvatars` store to prevent this? I'm not sure how it's all structured but we have a `hasRegistered` value on profiles. It requires its own node call to fetch though 😬 

https://github.com/hachiojidev/pancake-frontend/blob/e6edf4816f5d1d5f94fbc1d98437eccc1fea2cca/src/state/profile/helpers.ts#L73

Reply:
yeah it could be done, but it would require some refactoring in the profile code that _might_ be breaking 🥲 
Wouldn't want to get stuck in it...

Reply:
I'm actually torn on this one man. I think deactivated accounts (username but no nft) are going to be much less common than never-activated accounts (no username, no nft). And it's the second, more common occurrence, that is gonna fire network requests that 404.

I'd probably return this check to as it was previously until we have a reliable way of identifying the deactivated account scenario.

Reply:
actually one sec, it seems that `getProfileAvatar` is not used anywhere else....